doc/classes/Array.xml

```diff
@@ -648,28 +673,44 @@
 			<return type="bool" />
 			<param index="0" name="right" type="Array" />
 			<description>
-				Compares the left operand [Array] against the [param right] [Array]. Returns [code]true[/code] if the sizes or contents of the arrays are [i]not[/i] equal, [code]false[/code] otherwise.
+				Returns [code]true[/code] if the array's size or its elements are different than [param right]'s.
 			</description>
 		</operator>
 		<operator name="operator +">
 			<return type="Array" />
 			<param index="0" name="right" type="Array" />
 			<description>
-				Concatenates two [Array]s together, with the [param right] [Array] being added to the end of the [Array] specified in the left operand. For example, [code][1, 2] + [3, 4][/code] results in [code][1, 2, 3, 4][/code].
+				Appends the [param right] array to the left operand, creating a new [Array]. This is also known as an array concatenation.
+				[codeblocks]
+				[gdscript]
+				var array1 = ["One", 2]
+				var array2 = [3, "Four"]
+				print(array1 + array2) # Prints ["One", 2, 3, "Four"]
+				[/gdscript]
+				[csharp]
+				// Note that concatenation is not possible with C#'s native Array type.
+				var array1 = new Godot.Collections.Array{"One", 2};
+				var array2 = new Godot.Collections.Array{3, "Four"};
+				GD.Print(array1 + array2); // Prints ["One", 2, 3, "Four"]
+				[/csharp]
+				[/codeblocks]
+				[b]Note:[/b] For existing arrays, [method append_array] is much more efficient than concatenation and assignment with the [code]+=[/code] operator.
 			</description>
 		</operator>
 		<operator name="operator &lt;">
 			<return type="bool" />
 			<param index="0" name="right" type="Array" />
 			<description>
-				Performs a comparison for each index between the left operand [Array] and the [param right] [Array], considering the highest common index of both arrays for this comparison: Returns [code]true[/code] on the first occurrence of an element that is less, or [code]false[/code] if the element is greater. Note that depending on the type of data stored, this function may be recursive. If all elements are equal, it compares the length of both arrays and returns [code]false[/code] if the left operand [Array] has fewer elements, otherwise it returns [code]true[/code].
+				Compares the elements of both arrays in order, starting from index [code]0[/code] and ending on the last index in common between both arrays. For each pair of elements, returns [code]true[/code] if this array's element is lower than [param right]'s, [code]false[/code] if this element is greater. Otherwise, continues to the next pair.
```

Comment:
```suggestion
				Compares the elements of both arrays in order, starting from index [code]0[/code] and ending on the last index in common between both arrays. For each pair of elements, returns [code]true[/code] if this array's element is less than [param right]'s, [code]false[/code] if this element is greater. Otherwise, continues to the next pair.
```
Same elsewhere

Reply:
Replaced every instance of lower with less.